app/Models/Bible/Bible.php

```diff
@@ -435,18 +435,17 @@ public function scopeIsContentAvailable(Builder $query, Collection $access_group
     {
         return $query->whereExists(function ($query) use ($access_group_ids) {
             return $query->select(\DB::raw(1))
-                ->from('access_group_filesets as agf')
-                ->join('bible_fileset_connections as bfc', 'agf.hash_id', 'bfc.hash_id')
+                ->from('sys_license_group_access_groups_view as agf')
```

Comment:
rename from agf to lgag so it does not confuse. Same comment for other such instances

Reply:
@bradflood I have changed it according to your recommendation. You can view here: https://github.com/faithcomesbyhearing/dbp/pull/1008/files#diff-8034f9e5f0a067a8790fa0344d5d1b6243cd75241e84f37c3ed55d9444ffa5f0R438